elements/pfe-page-status/src/pfe-page-status.scss

```diff
@@ -0,0 +1,78 @@
+@import "../../pfe-sass/pfe-sass";
+:host {
+  --pfe-status--bg-color: #{pfe-color(feedback--default)};
+  --pfe-status--text-color: #{pfe-color(feedback--default--lightest)};
+
+  display: block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+.flag {
+  position: fixed;
+  right: 0;
+  top: 0;
+  background: var(--pfe-status--bg-color);
```

Comment:
Can we use the more specific background-color just in case someone wants to add a texture to it for some silly reason?